packages/ripple-binary-codec/src/binary.ts

```diff
@@ -66,8 +86,9 @@ function serializeObject(object: JsonObject, opts: OptionObject = {}): Buffer {
   const filter = signingFieldsOnly
     ? (f: FieldInstance): boolean => f.isSigningField
     : undefined
-
-  coreTypes.STObject.from(object, filter).toBytesSink(bytesList)
+  ;(coreTypes.STObject as typeof STObject)
```

Comment:
Can this cast be returned to how it was previously?

Reply:
Unfortunately not - the `STObject` `.from` method is unique because it has dynamic parsing, whereas the other entries in `coreTypes` are concrete implementations so don't need to know about `definitions.json` when decoding.

That means that in order to access the special `.from` we have to cast down to `typeof STObject` instead of `typeof SerializedType`